README.md

```diff
@@ -1,2 +1,97 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # TO-DO-LIST
-It simply lists the things that you need to do and allows you to mark them as complete, and it uses ES6 and Webpack.
+
+> Description the project.
+- It simply lists the things that you need to do and allows you to mark them as complete, and it uses ES6 and Webpack.
+- Single page website.
+- configure with linters.
+
+## Screenshot
+
+![readme](./assets/screenshoots/M1.png)
+![readme](./assets/screenshoots/M2.png)
+![readme](./assets/screenshoots/D1.png)
+![readme](./assets/screenshoots/D2.png)
+
+# Live Demo
+
+[Live Demo Link](https://reemmohamedabdelfatah.github.io/TO-DO-LIST/)
+
+
+# Built With
+Major languages: HTML-CSS-JS
+Frameworks: NodeJS, Webpack, Es6 Modules
+Technologies used: VSCode, GitHub, Linter
+
+
+## Getting Started
+To get a local copy, follow these simple steps.
+
+### Prerequisites
+Web browser installed
+VSCode to be installed.
+GitHub account.
+Even Git Bash or VSCode Source of Control to clone it in your local machine.
+
+### Setup
+Cloning a repository
+* on Github.com, navigate to the main page of the repository;
+* Above the list of files, click "Code" button; 
+* Copy the URL to clone the repository.
+* Open Terminal;
+* Change the current working directory to the location where you want the cloned directory; 
+* Type git clone, and then paste the URL you copied earlier; 
+* Install Linters(linters.yml; .hintrc; .stylelintrc.json) - that linters configuration for Web, HTML and CSS
+
+# Install
+
+node.js
+npm
+linter configuration
+
+# Usage
+
+Browser
+VSCode
+GitHub
+
+# Run tests
+
+- follow this link :
+ [HTML, CSS and Javascript Linters link]( https://github.com/microverseinc/linters-config/tree/master/html-css-js)
+
+linters.yml for all of linters. (See below)
+Styelint for auto-correct CSS errors or fixing the CSS style.
+WebHint for cross-browser compatibility, speed, and to improve your Web's accessibility.
+Eslint for correct javascript files.
+
+## Authors
+
+👤 **Reem Muhammed**
+
+- GitHub: [@github](https://github.com/ReemMohamedAbdelfatah)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/reem-abd-el-fatah-a07543116)
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+- Design
+Credit goes to **Cindy Shin's** design on [Behance] (https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). This is a mockup of Cindy's design.
```

Comment:
- [ ] I noticed that this information is not useful and doesn't have to be here please remove it and make sure to remove all unnecessary information from your readme to make it more professional  :+1: